apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/UpcomingInvoice.tsx

```diff
@@ -195,33 +232,39 @@ const UpcomingInvoice = ({ slug }: UpcomingInvoiceProps) => {
                 </Collapsible>
               ))}
 
+            {computeCredits && (
+              <tbody>
+                <tr className="border-b">
+                  <td className="py-2 text-sm max-w-[200px]">
+                    <span className="mr-2">{computeCredits.description}</span>
+                    <InvoiceTooltip
+                      text="Paid plans come with $10 in Compute Credits to cover one Starter instance or parts of any other instance. Compute Credits are given to you not only for the first month but for every month while you are on a paid plan."
```

Comment:
```suggestion
                      text="Paid plans come with $10 in Compute Credits to cover one Starter instance or parts of any other instance. Compute Credits are given to you every month and do not stack up while you are on a paid plan."
```

> not only for the first month but for every month

sounds a bit repeated and there's easier ways to put it, no hard feelings though